### ISSUE TITLE
Intent/Trial emails need a TL;DR

### ISSUE THREAD

**Issue by jyasskin** [GoogleChrome/chromium-dashboard#1718]
### Is your feature request related to a problem? Please describe.

The email drafted by https://chromestatus.com/admin/features/launch/6438627087220736/2?intent=1 buries the lede 1.5 pages down the email. People reading the email need to know how to try out the feature (the "DevTrial instructions" field), so that should appear first, and ideally not even behind a link.

### Describe the solution you'd like

Order the fields in the generated email by their importance at this stage of the process. For example, I2P might put the explainer first, Ready for Trial and I2E might put the instructions for enabling the feature or signing up for the origin trial first, and I2S might put the specification, risks, and TAG review first.

**Comment by samuelgoto** [comment]
Agreed on the importance of the instructions to be put up and center before other details.
